golang/build.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: golang-build
+spec:
+  inputs:
+    params:
+    - name: package
+      description: base package to build in
+    - name: packages
+      description: "packages to build (default: ./cmd/...)"
```

Comment:
I don't think we should document the default in the description, users should be able to get information about the default from (hopefully someday generated) documentation. Having to duplicate it makes it easy to have them fall out of sync.

Reply:
oh yeah :sweat_smile:

Reply:
Ping, I still see documented defaults in the latest commit